### ISSUE TITLE
At the test stage, I want to input the real image or not random of noise to test.

### ISSUE THREAD

**Issue by maxin19940317** [eriklindernoren/Keras-GAN#202]
I already finished the stage of the train. Now, I'm going to restore my real picture, Can I put the real picture into the test?
Thank you!

**Comment by hiteshnitetc** [comment]
Yes you can, but make sure, you didn't inset these images during training time, this will give you true evaluations.
